packages/react-sdk/src/core/components/CallLayout/SpeakerLayout.tsx

```diff
@@ -32,6 +33,11 @@ export type SpeakerLayoutProps = {
    * Providing `null` will hide the bar.
    */
   participantsBarPosition?: 'top' | 'bottom' | 'left' | 'right' | null;
+  /**
+   * Hard limits the number of the participants rendered in the participants bar.
+   * Providing string `dynamic` will calculate hard limit based on screen width/height.
+   */
+  participantsBarLimit?: 'dynamic' | number;
```

Comment:
Why not make this an optional number? I think it is already used like that. By default it is `undefined` but when provided, we use the value. I don't think we need a `dynamic` as a value here.

Reply:
The functionality can reach three states - without the limit (undefined) - it's used with the normal - scrolling behaviour where no limitation is applied, number to limit it to that specific amount and "dynamic" to calculate the amount based on the screen width/height which will render maximal amount that fits into the parent and does not trigger scrolling behavior. Optional number would mean to limit it only to two of the aforemention behaviours.